Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/Beckn/Confirm.hs

```diff
@@ -74,7 +74,13 @@ confirm transporterId (SignatureAuthResult _ subscriber) req =
                   void $
                     BP.sendRideAssignedUpdateToBAP dConfirmRes.booking ride
               DS.driverScoreEventHandler DST.OnNewRideAssigned {merchantId = transporterId, driverId = Id driverId}
-            _ -> do
+            DConfirm.DConfirmResRentalBooking -> do
```

Comment:
why this duplicated code added?

Reply:
@0utkarsh mentioned it might be unsafe that why